htmlreport/cppcheck-htmlreport

```diff
@@ -394,9 +394,10 @@ if __name__ == '__main__':
     parser.add_option('--title', dest='title',
                       help='The title of the project.',
                       default='[project name]')
-    parser.add_option('--file', dest='file',
+    parser.add_option('--file', dest='file', action="append",
                       help='The cppcheck xml output file to read defects '
-                           'from. Default is reading from stdin.')
+                           'from. Multiple times could be defined. '
```

Comment:
I am not a good technical writer but I think `Multiple times could be defined.` sounds strange. Maybe `You can combine results from several xml reports i.e. '--file file1.xml --file file2.xml ..'`.